server/src/main/java/io/druid/indexing/overlord/supervisor/SupervisorSpec.java

```diff
@@ -31,4 +33,6 @@
   String getId();
 
   Supervisor createSupervisor();
+
+  List<String> getDataSources();
```

Comment:
@pjain1 @himanshug would you elaborate more on this? Do you have a supervisor indexing multiple dataSources? Would you tell me what use case needs that supervisor?

Reply:
don't remember exactly the reason for the list, may be it was for future proofing. If I am not wrong it was an ask from @himanshug, so may be he can explain better.

Reply:
@pjain1 thanks. I'm implementing a new supervisor, and would like to add some javadoc to reduce confusion. Any comments will be much helpful.